src/Maker/MakeFunctionalTest.php

```diff
@@ -55,6 +56,7 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             'test/Functional.tpl.php',
             [
                 'web_assertions_are_available' => trait_exists(WebTestAssertionsTrait::class),
+                'panther_is_available' => class_exists(PantherTestCaseTrait::class),
```

Comment:
Ah! one more thing: this needs to use `trait_exists()`.

Reply:
resolved !